### ISSUE TITLE
4.034 fails to build: error: use of undeclared identifier 'WIFEXITED'

### ISSUE THREAD

**Issue by yurivict** [verilator/verilator#2351]
```
../V3Os.cpp:349:17: error: use of undeclared identifier 'WIFEXITED' 
        UASSERT(WIFEXITED(ret), "system(" << command << ") returned unexpected value of " << ret);
                ^
../V3Os.cpp:350:31: error: use of undeclared identifier 'WEXITSTATUS'
        const int exit_code = WEXITSTATUS(ret);
                              ^
2 errors generated.
```

**Comment by yurivict** [comment]
```#include <sys/wait.h>``` is missing.

**Comment by yurivict** [comment]
https://github.com/verilator/verilator/pull/2353

**Comment by wsnyder** [comment]
Thanks, fixed with #2353
